os/kernel/debug/sysdbg.c

```diff
@@ -208,20 +208,27 @@ static void simulate_undef_abort(void)
 
 static void show_usage(void)
 {
-	dbg("USAGE:\n");
-	dbg("sysdbg enable_monitor  /* Allocates memory and enables monitoring */\n");
-	dbg("sysdbg disable_monitor /* Frees the memory and disables monitoring */\n");
-	dbg("sysdbg read   /* Reads /dev/sysdbg to view debug information*/\n");
-	dbg("sysdbg max_task_count <count> /* Enter max_task_count */\n");
-	dbg("sysdbg max_irq_count <count> /* Enter max_irq_count */\n");
-	dbg("sysdbg max_sem_count <count> /* Enter max_sem_count */\n");
-	dbg("sysdbg data_abort /* Simulate data abort */\n");
-	dbg("sysdbg prefetch_abort /* Simulate Prefetch abort */\n");
-	dbg("sysdbg undef_abort  /* Simulate Undefined abort */\n");
-	dbg("sysdbg usr_assert /* Simulate User Assert */\n");
-	dbg("sysdbg dump_stack /* Dumps the current task stack */\n");
-	dbg("sysdbg dump_allstack /* Dumps the stack of all tasks */\n");
-	dbg("sysdbg help /* Display the help */\n");
+	dbg("\n");
+	dbg(" ______________________________________________________________________________\n");
+	dbg("|USAGE: sysdbg [Option] [Count]\n");
+	dbg("|  Option       -     Description \n");
+	dbg("|______________________________________________________________________________\n");
+	dbg("|     1         -  enable_monitor  : Allocates memory for debug struct & enables sysdbg monitoring\n");
+	dbg("|     2         -  disable_monitor : Frees the memory for debug struct & disables sysdbg monitoring\n");
+	dbg("|     3         -  read            : Reads /dev/sysdbg to view saved debug information\n");
+	dbg("|     4 [count] -  max_task_count  : Updates the maximum count value for task monitoring\n");
+	dbg("|                                    count - input value in the power of 2\n");
+	dbg("|     5 [count] -  max_irq_count   : Updates the maximum count value for irq monitoring\n");
+	dbg("|                                    count - input value in the power of 2\n");
+	dbg("|     6 [count] -  max_sem_count   : Updates the maximum count value for semaphore monitoring\n");
+	dbg("|                                    count - input value in the power of 2\n");
+	dbg("|     7         -  data_abort      : Simulates the data abort with null pointer access\n");
```

Comment:
Is there any reason sysdbg has abort simulations?

Reply:
I think we should remove. Because this is not debugging and there is another example for this like mpu test and reboot reason test.